qos/qos_rule_test.go

```diff
@@ -63,8 +64,7 @@ func TestBuildQosRules(t *testing.T) {
 			0x1, 0x1, 0xff, 0xff, 0xff, 0xff, 0x50, 0x3, 0xe8, 0x11, 0x2, 0x2, 0x2,
 			0x2, 0xff, 0xff, 0xff, 0xff, 0x40, 0x7, 0xd0, 0x32, 0x18, 0x10, 0x3, 0x3,
 			0x3, 0x3, 0xff, 0xff, 0xff, 0xff, 0x50, 0xb, 0xb8, 0x11, 0x4, 0x4, 0x4,
-			0x4, 0xff, 0xff, 0xff, 0xff, 0x40, 0xf, 0xa0, 0xc8, 0x5, 0xff, 0x0, 0x6,
-			0x31, 0xff, 0x1, 0x1, 0xff, 0x8}
+			0x4, 0xff, 0xff, 0xff, 0xff, 0x40, 0xf, 0xa0, 0xc8, 0x0}
```

Comment:
Same comment here, can you explain a bit what was wrong with this test?

Reply:
QosID needs to be integer and it was string before. This is fixed. Besides QosFlow has 2 rules stored stored in a map and depending on which rule is first processed, expected output  changed. This problem is fixed in the tests.